### DISCUSSION TITLE
Privacy issues

### DISCUSSION THREAD

**Discussion by sfraysse** [gaia-x-dases/xapi-virtual-classroom#11] [General]
## Private messages

I would remove the `posted private message` statement from the spec.

If these messages are private, this means that users think they are speaking in a private context, and that their conversations are not recorded. So tracking private messages would rise a big issue with the GDPR.

## Public messages

`posted public message` : in my opinion, recording public messages is acceptable but we should think about privacy issues and give recommendations in our **human readable spec**.

I think about 3 scenarios that should be supported be this profile :

- **Content of messages is included in the statements:** in this case, participants of the virtual classroom should be informed that their conversations are recorded, explain why and obtain their consent.

- **Content of messages is NOT included in the statements:** such statements (without messages) are useful from a learning analytics perspective because they give an idea of who participated actively, without divulging what they said.

- **Anonymization of actors:** the actor may be anonymized and the message may be included. This opens the door of text analysis, without compromising the identity of the authors. However, users should be informed that their conversations are recorded, explain why and obtain their consent. This is important because the content of the messages may include names of users such as *"John, regarding your question, ..."*.

**Comment by quitterie-lcs** [comment]
I did not consider that posted a private message in a chat could have issue in the GDPR. I am getting more information on this issue with the DPO at FUN. 
But in this way, it means that we consider the profile to be used in Europe and the adoption would not be international. What do you think of that ?

**Reply by sfraysse** [reply]
I think being applicable to Europe is a priority.
And showing that we are concerned by privacy issues would be valuable, even for non-European audiences.

From a vendor perspective, taking care about privacy does to close the international market.
It opens the European market, which is part of the international market.

Now, people are always free to add more things in their own implementation.

My point is not to prohibit such statements, but just to let them out of the scope of the spec if we think they may be problematic.
